tests/__init__.py

```diff
@@ -1,5 +1,3 @@
-# -*- coding: utf-8 -*-
-
```

Comment:
Maybe replace `__auther__` , `__email__` to team identifiers/email if you want to keep these attributes.